datumaro/components/dataset_base.py

```diff
@@ -87,7 +87,11 @@ def __init__(
             media = point_cloud
 
         self.__attrs_init__(
-            id=id, subset=subset, media=media, annotations=annotations, attributes=attributes
+            id=id,
+            subset=subset,
+            media=media,
+            annotations=annotations,
+            attributes=attributes,
```

Comment:
Please revert those changes which are unrelated to this feature.

Reply:
Fix.